packages/block-library/src/navigation/edit/index.js

```diff
@@ -61,6 +61,15 @@ import useCreateNavigationMenu from './use-create-navigation-menu';
 import { useInnerBlocks } from './use-inner-blocks';
 import { detectColors } from './utils';
 
+const navigationStorage = window.localStorage;
+
+if ( ! navigationStorage.getItem( 'nav_menus_created' ) ) {
+	navigationStorage.setItem(
+		'nav_menus_created',
```

Comment:
I think this is more like `classic_menus_imported` unless I am reading the code wrong?

Reply:
Yep, I thought about similar cases that could happen with block menus, so put the name for every type. But seems this only will happen with classic ones.